CIP-governance-metadata-cc-rationale/README.md

```diff
@@ -0,0 +1,133 @@
+---
+CIP: ?
+Title: Governance metadata - Constitutional Committee vote rationale
+Category: Metadata
+Status: Proposed
+Authors:
+    - Ryan Williams <ryan.williams@intersectmbo.org>
+    - Eystein Magnus Hansen <eysteinsofus@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-07-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artifacts.
+Some of these artifacts support the linking off-chain metadata, as a way to provide context.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata should be formed and handled.
+But this is intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for off-chain metadata vocabulary that can be used to give context to Constitutional Committee votes.
+
+## Motivation: why is this CIP necessary?
```

Comment:
Breaking it down to the bare minimum of why this CIP is necessary is to develop a standard for how the individual ICC/CC members should communicate their voting rationale to the Cardano Community.

I'm not convinced that we need to define "to show that the CC is fair and reasonable" in the standard. The standard should not define the content, in my opinion.

Reply:
> I'm not convinced that we need to define "to show that the CC is fair and reasonable" in the standard. The standard should not define the content, in my opinion.

In the motivation, I am articulating the problem(s) that the proposal aims to solve per stakeholder group.
Its not a part of the standard, but it is a motivation for writing the standard.

If there is no metadata standard for CC vote rationale, we could end up in a situation where little to no rationale is attached to votes which I feel would be a problem.
This is because, without the contextual rich metadata, I feel it would be very hard to judge CC's "fairness" in their votes.

Reply:
There are many reasons for CC to publish their rationale. Fair is one but also in the case of dissent it can actually help show the community that this is something the CC is also divided on and the rationales provided can help in the debate the larger community is having. This I feel is one of the more underrated but hugely important role of CC or as the quote goes "Governance is a dumpster fire, good governance is when you contain the fire in the dumpster." (not sure we should write that in the motivation haha). More generally it is to show the diverse interpretation of the constitution, but also what unifies and creates stable rules for the users of the blockchain. If someone interested in Cardano and want to know how the constitution is practiced they can go to the library of rationales to get a general idea of practice of interpretation.  I would also say fair but also accountability as it will likely be the main way that you judge if you have confidence in the CC or not.

Reply:
I am with @thenic95 on this.  The CIP is not about "why" rationales are published, it is simply a recommended "how". As it isn't enforceable (rationales could be published differently), we need to describe what benefit there would be to CC members in using this standard.

Some thoughts from me are:
- Allows rationales to be machine-readable (e.g. AI training)
- Allows rationale sections to be queryable
- Provides a structured breakdown so it is easier to maintain contexts when translating rationales

Reply:
>  The CIP is not about "why" rationales are published

The CIP is not about "why", but the CIP's motivation is about the "why"

> Some thoughts from me are:
> 
> Allows rationales to be machine-readable (e.g. AI training)
> Allows rationale sections to be queryable
> Provides a structured breakdown so it is easier to maintain contexts when translating rationales

Good points, I have added a note for this